### PR TITLE
FXSD-257/base

### PR DESCRIPTION
**Pull request by duduneto**
### PR Summary: 

This PR will include a new homepage template and 3 additional template pages (Furniture Maker, Sign Maker, General Maker). Also a new layout for the industries theme is created.

### Why are these changes introduced?

Homepage, Furniture Maker, Sign Maker, General Maker template pages.

### Visual impact on existing themes
Will add a `Industries` layout to the current theme.

### Demo links
`Development Env`

- [Store](https://foxbox-inventables-dev.myshopify.com/)
- [Editor](https://admin.shopify.com/store/foxbox-inventables-dev/themes/159211127078/editor)

### CODE REVIEW

File: layout/industries.liquid [hunk 1]
```diff
@@ -0,0 +1,601 @@
...
+    {% if CFH contains 'admin_bar_iframe' %}
+      {% assign admin = true %}
+    {% elsif CFH contains 'preview_bar_injector-' %}
+      {% assign admin = true %}
+    {% elsif CFH contains 'Online store editor' %}
+      {% assign admin = true %}
+    {% elsif CFH contains 'designMode' %}
+      {% assign admin = true %}
+    {% endif %}
+
```
Comment (thomasjlee) [inline layout/industries.liquid:16]: Please remove these lines. We no longer need `CFH`.

**Review by thomasjlee — CHANGES_REQUESTED** [review]
Hi @duduneto, this is looking good!

I added a comment about the `CFH` variable in `industries.liquid`.

Also I noticed the main navigation is no longer sticky. It should behave like this:

https://github.com/inventables/ecomm-theme-dawn/assets/26750330/b27e6521-4437-4813-9aae-a483a811c47c

File: layout/industries.liquid (hunk 1 above)
Comment (duduneto) [reply layout/industries.liquid]: Solved!

**Comment by natedemery** [comment]
@duduneto can you rebase this PR? There are some merge conflicts.

**Review by thomasjlee — APPROVED** [review]

**Review by natedemery — APPROVED** [review]

**Pull request merged**